clang-tools-extra/clang-doc/Serialize.cpp

```diff
@@ -525,7 +525,13 @@ template <typename T>
 static void populateInfo(Info &I, const T *D, const FullComment *C,
                          bool &IsInAnonymousNamespace) {
   I.USR = getUSRForDecl(D);
-  I.Name = D->getNameAsString();
+  auto ConversionDecl = dyn_cast_or_null<CXXConversionDecl>(D);
+  if (ConversionDecl && ConversionDecl->getConversionType()
+                            .getTypePtr()
+                            ->isTemplateTypeParmType())
```

Comment:
```suggestion
  if(auto ConversionDecl = dyn_cast_or_null<CXXConversionDecl>(D); ConversionDecl && ConversionDecl->getConversionType()
                            .getTypePtr()
                            ->isTemplateTypeParmType())
```
I think you can rewrite this as a single conditional. https://godbolt.org/z/b46c96ccr